src/Symfony/Component/HttpKernel/HttpCache/Esi.php

```diff
@@ -84,7 +84,7 @@ public function hasSurrogateCapability(Request $request)
     public function addSurrogateCapability(Request $request)
     {
         $current = $request->headers->get('Surrogate-Capability');
-        $new = 'symfony2="ESI/1.0"';
+        $new = 'symfony3="ESI/1.0"';
```

Comment:
I don't exactly know, I was just wondering. But if an application is strictly comparing the header value, then it will require a change in the application code or in the reverse proxy configuration I suppose. But as this is from the built-in HttpCache, it's not supposed to be used in production, so the impact of breaking something should be very small I guess.